tutorials/firstcodemod.md

```diff
@@ -155,6 +155,14 @@ namespace Celeste.Mod.Example {
 }
 ```
 
+### Hooking Methods
+There are currently 3 ways to hook methods. These are:
+
+* RuntimeDetour: This is currently used by [RainbowMod](https://github.com/EverestAPI/RainbowMod) and [MadelineEnergySelector](https://github.com/EverestAPI/MadelineEnergySelector), and will be deprecated for end-user mods once HookedMethod has a stable version.
```

Comment:
I know what you're trying to say with "deprecated for end-user mods", but it sounds misleading. Maybe state that it's "too low-level" and "not suited for Celeste modding", while still listing "HookedMethod" as the user-friendly replacement.  
After all, RuntimeDetour is an actively maintained component in MonoMod. If someone wants to mod Celeste and use it for whatever reason, then finds a bug, I'm going to fix it. "Deprecated" implies "end of life", which isn't the case.